tests/framework/utils/k8s_helper.go

```diff
@@ -936,3 +1044,14 @@ func (k8sh *K8sHelper) DeleteRoleAndBindings(name, namespace string) error {
 
 	return nil
 }
+
+func (k8sh *K8sHelper) ScaleStatefulSet(statefulsetName string, replicationSize int) bool {
```

Comment:
is the statefulset ever in a non-default namespace? to keep it clean between test suites, it would be good to use different namespaces